controllers/datafeed_controller.py

```diff
@@ -107,9 +107,20 @@ class FMSAPIEventAlliancesEnqueue(webapp.RequestHandler):
     Handles enqueing getting alliances from the FMS API
     """
     def get(self, when):
+        events = skipped_events = []
         if when == "now":
             events = EventHelper.getEventsWithinADay()
             events = filter(lambda e: e.official, events)
+
+            # Check if we should do a full query, or temporarily skip events
+            # that are not ending today. (All events will be checked on
+            # half-hour intervals, and events ending today whenever the cron
+            # job runs.)
+            minutes_from_full = (((int(time.time()) / 60) + 15) % 30) - 15
```

Comment:
I don't totally understand the logic here. Is the intent to check all events every 30 minutes, and only events ending today every 5?

Would it be clearer to split into two cronjobs, each with more specific jobs and less logic?